scipy/optimize/_trustregion_constr/minimize_trustregion_constr.py

```diff
@@ -0,0 +1,521 @@
+    x : ndarray, shape (n,)
+        Solution found.
+    fun : float
+        Objective function at the solution.
+    grad : ndarray, shape (n,)
+        Gradient of the objective function at the solution.
+    lagrangian_grad : ndarray, shape (n,)
+        Gradient of the Lagrangian function at the solution.
+    nfev : integer
+        Number of objective function evaluations.
+    ngev : integer
+        Number of gradient evaluations. Will be zero when
+        using finite-differences approximations.
+    nhev : integer
+        Number of Hessian evaluations. Will be zero when
+        using finite-differences or quasi-Newton approximations.
+    niter : int
+        Total number of iterations.
+    cg_niter : int
+        Total number of CG iterations.
+    cg_stop_cond : int
+        Reason for CG subproblem termination at last iteration:
+
+            * 0 : CG subproblem not evaluated;
+            * 1 : Iteration limit was reached;
+            * 2 : Reached the trust-region boundary;
+            * 3 : Negative curvature detected;
+            * 4 : Tolerance was satisfied.
+
+    execution_time : float
+        Total execution time.
+    trust_radius : float
+        Trust radius at the last iteration.
+    penalty : float
+        Penalty function at last iteration.
+    tolerance : float
+        Tolerance for barrier subproblem at the last iteration.
+        Exclusive for 'tr_interior_point'.
+    barrier_parameter : float
+        Barrier parameter at the last iteration. Exclusive for
+        'tr_interior_point'.
+    status : {0, 1, 2, 3}
+        Termination status:
+
+            * 0 : The maximum number of function evaluations is exceeded.
+            * 1 : `gtol` termination condition is satisfied.
+            * 2 : `xtol` termination condition is satisfied.
+            * 3 : `callback` function requested termination.
+
+    message : str
+        Termination message.
+    method : {'equality_constrained_sqp', 'tr_interior_point'}
+        Optimization method used.
+    constr_violation : float
+        Maximum constraint violation at the solution.
+    optimality : float
+        Infinity norm of the Lagrangian gradient at the solution.
+    constr : list of ndarray
+        List of constraint values at the solution. This list will give
+        the values in the same order used  in `constraints`. When bound
+        constraints are present one extra element will be include(in
+        the end of the list) to account to the values corresponding
+        to these constraints
+    jac : list of {ndarray, sparse matrix}
+        List of constraints Jacobian matrices evaluated at the solution.
+        This list will give the values in the same order used in
+        `constraints`. When bound constraints are present one extra element
+        will be include (in the end of the list) to account to the values
+        of these constraints Jacobian matrices.
+    v : list of ndarray
```

Comment:
I think it needs some clarification for the case of interval constraint.

It seems to me that `v < 0` indicates that the upper bound is active and `v > 0` that lower bound is active. Am I correct here? It should be included into the doc.

Considering this I believe it makes sense to include `Bounds`, to indicate which is active and which is not. But perhaps a clearer approach is to include them separately (well, maybe not).

Reply:
Actually is the oposite :) ``v > 0`` for an active upper bound and ``v<0`` for an active lower bound. I just added a note about it.

Besides that, I think it is better to keep it as it is, without separating them.

Reply:
I see now, there are several places where the sign changes compared to "Numerical Optimization".